helper/storagepacker/storagepacker_test.go

```diff
@@ -237,3 +237,56 @@ func TestStoragePacker_DeleteMultiple(t *testing.T) {
 		}
 	}
 }
+
+func TestStoragePacker_DeleteMultiple_ALL(t *testing.T) {
+	storagePacker, err := NewStoragePacker(&logical.InmemStorage{}, log.New(&log.LoggerOptions{Name: "storagepackertest"}), "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	ctx := context.Background()
+
+	// Persist a storage entry
+	itemsToDelete := make([]string, 0, 10000)
```

Comment:
It might be a good idea to have a different `for` loop before this that also creates a bunch of items. After the deletion call later, we might want to see if these entries are untouched.

Reply:
I think the other test in this file should cover that case. Since it only deletes half of the values. Let me know if you think we still need an update here though.

Reply:
Sounds good.